docs/config-openshift-v4_13.md

```diff
@@ -1,24 +1,22 @@
 ---
-title: OpenShift v4.13.0-experimental
+title: OpenShift v4.13.0
 parent: Configuration specifications
 nav_order: 150
```

Comment:
Needs update.

Reply:
Oh do we update the nav order each time? sorry I am not familiar with that, do I just ++ or somthing?

Reply:
Ah I see, so 144 would be appropriate, and I assume the exp spec has the highest number for the set?

Reply:
It needs to be one less than the previous stable spec, same rule as a normal stabilization.